server/emulator.go

```diff
@@ -65,6 +65,8 @@ func NewEmulatorAPIServer(server *EmulatorServer, backend *backend.Backend, stor
 
 	router.HandleFunc("/emulator/config", r.Config)
 
+	router.HandleFunc("/emulator/codeCoverage", r.CodeCoverage).Methods("GET")
+
 	return r
```

Comment:
nit: would it be possible to also add a method to reset coverage report ? It can be very useful.

Reply:
That is quite a nice idea :100: :100: :100: 
I will add it for sure!!!

Reply:
@bluesign With the current API, we can achieve this with something like:

```go
func (b *Blockchain) ResetCoverageReport() {
	b.vmCtx.ReusableCadenceRuntimePool.CoverageReport = runtime.NewCoverageReport()
	b.coverageReport = b.vmCtx.ReusableCadenceRuntimePool.CoverageReport
}
```

I am not a big fan of reaching this with `b.vmCtx.ReusableCadenceRuntimePool.CoverageReport`, however, the `CoverageReport` lacks a `Reset` method, for the time being.

What are your thoughts on this? Do we first add the required API in `CoverageReport`, or we add a workaround like above, and come back to fix it later on?

Reply:
I have added here (https://github.com/onflow/flow-emulator/pull/332/commits/4e7b422e36c101f156ae74efa40ca262bd0958f6) the workaround, for reference.